10_Examples/jett-onboarding/src/talking.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TALKING_H_
+#define TALKING_H_
+
+#include <iostream>
+#include "string.h"
+
+using namespace std;
+
+class talking
+{
+public:
+  int num1;
+
+  talking();    // constructor
+  ~talking();   // destructor
+
+  string talk(string words);
```

Comment:
In general, we can split any code project into source files (.cpp), header files (.hpp), and tests. The header file is usually where you put all the nice comments explaining what functions do, what assumptions they make, if they throw errors, etc.

Well written header files and tests are often all the documentation you might get.